### ISSUE TITLE
python-requests <= 2.8.0?

### ISSUE THREAD

**Issue by lilmike** [mopidy/mopidy-gmusic#113]
Hi,
I tried using the gmusic mopidy addon in arch, and it said that my version of requests (2.9.1) was too high, from the error message. I have python-requests installed from the arch linux repositories, and also tried installing with pip2, as that seems to have version 1.6.6, which I think would work. However, it seems the system-provided python-requests it used first, and I can't remove it because most of the mopidy packages I have installed depend on it.
 Any idea how to solve this?
Thanks,
-Michael.

**Comment by jodal** [comment]
There's multiple dependencies on requests within Mopidy-GMusic's tree of dependencies. If you run `mopidy deps` you can inspect the dependencies and find that:

- Mopidy-GMusic depends on:
  - requests>=2.0
  - Mopidy, which depends on:
    - requests>=2.0
  - gmusicapi, which depends on:
    - requests!=1.2.0,!=2.2.1,<2.8.0,>=1.1.0
    - gpsoauth, which depends on:
      - requests without any version specifier
    - MechanicalSoup, which depends on:
      - requests>=2.0

In summary, the project to blame for the issue here is gmusicapi. Actually, it seems gmusicapi fixed this in commit simon-weber/gmusicapi@8d222317bc3c5daa08cdfcb4d0b65ebf59271dc8, but the fix hasn't been released yet.

**Comment by jjok** [comment]
@jodal Did you see that there's a version [10.0.0 of gmusicapi](https://github.com/simon-weber/gmusicapi/releases/tag/10.0.0)?

**Comment by jodal** [comment]
Great, then this is properly fixed :-)

**Comment by jjok** [comment]
I just installed the current version of mopidy-gmusic and it came with the latest gmusicapi.
